modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/web/internal/layout/display/page/test/JournalArticleLayoutDisplayPageProviderTest.java

```diff
@@ -69,6 +75,48 @@ public void tearDown() throws Exception {
 		ServiceContextThreadLocal.popServiceContext();
 	}
 
+	@Test
+	public void testGetLayoutDisplayPageObjectProviderJournalArticleWithExpiredArticleVersionInfoItemReference()
+		throws Exception {
+
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext(_group.getGroupId());
+
+		JournalArticle originalArticle = JournalTestUtil.addArticle(
+			_group.getGroupId(),
+			JournalFolderConstants.DEFAULT_PARENT_FOLDER_ID);
+
+		JournalArticleLocalServiceUtil.updateArticle(
```

Comment:
Hi @bakayattila, to avoid any future issues it would be better to reassign the update to originalArticle.

Reply:
The whole point of the use case is that I have version 1.0 and version 1.1 after the update a new version is created if I override the original version with it I cannot reference the older version to test the issue.